src/test/java/org/quiltmc/launchermeta/version_manifest/VersionManifestTest.java

```diff
@@ -52,6 +70,15 @@ public void testParseFullJson() throws IOException {
         assertEquals(manifest.getLatestVersions().getRelease(), json.getAsJsonObject().get("latest").getAsJsonObject().get("release").getAsString(), "Size of version array matches json");
     }
 
+    @Test
+    public void testParseFullJsonV2() throws IOException {
```

Comment:
Should tests for V2 be their separate methods?

Reply:
I think making these parameterized tests is probably better. It would be weird for the `testParseTestJson` methods, but I think a public static final field for the expected manifest would work.

Reply:
Could you clarify a bit? I don't see how that would be weird. I'll try committing something and you tell me if it's good :+1:

Reply:
Weird as in I wasn't sure how it would work as i started writing the comment, but figured it out toward the end.